cpp/lib/include/Subfields.h

```diff
@@ -69,31 +94,36 @@ class Subfields {
     /** Swaps out all subfields' data whose subfield code is "subfield_code" and whose data value is "old_value". */
     void replace(const char subfield_code, const std::string &old_value, const std::string &new_value);
 
-    void erase(const char subfield_code) { subfield_code_to_data_map_.erase(subfield_code); }
+    void erase(const char subfield_code);
     void erase(const char subfield_code, const std::string &value);
```

Comment:
I think a comment would be great here.  Like "erases all subfields with subfield code "subfield_code" iff the subfield value is "value"."